start-localhost.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# These are only needed for dev
+echo "Running on Dev, setting up some test data"
+wait-port ${DJANGO_DB_HOST}:${DJANGO_DB_PORT} -t 15000
```

Comment:
line 5 and line 7 still exists in start.sh, which will be called later. Do we need to have both lines here?

Reply:
I have it in this script because it needs to wait to run the MySQL stuff here.

But this isn't run in production so it *might* have to wait there. We could take it out of start.sh and assume that production DB will always be running. This was *really* just there because of docker-compose so probably not even necessary to run in prod script.

Reply:
I removed it from start.sh